src/main/java/net/minecraftforge/fml/client/GuiModList.java

```diff
@@ -415,8 +423,18 @@ else if (disableable != Disableable.YES)
 
             IModGuiFactory guiFactory = FMLClientHandler.instance().getGuiFactoryFor(selectedMod);
             configModButton.visible = true;
-            configModButton.enabled = guiFactory != null && guiFactory.mainConfigGui(this) != null;
-
+            configModButton.enabled = false;
+            if(guiFactory != null)
```

Comment:
This is a concern, do we want things to initalize their screens just to check if they HAVE a screen? that needs to be fixed